github-bot/harvester_github_bot/backport.py

```diff
@@ -0,0 +1,113 @@
+from harvester_github_bot import zapi, repo, \
+    BACKPORT_LABEL_KEY
+
+import re
```

Comment:
let's move the regular library import to the top as it will be more consistent.

Reply:
updated